runtime/Cargo.toml

```diff
@@ -7,8 +7,8 @@ edition = "2021"
 crate-type = ["rlib", "cdylib", "staticlib"]
 
 [dependencies]
-cairo-felt = "0.8.5"
-cairo-lang-runner = "=2.2"
+cairo-felt = "0.8.2"
+cairo-lang-runner = "2.3.1"
```

Comment:
make it pinned to the minor (but not patch), e.g

`=2.3` this will allow cargo to choose any 2.3.x favoring newer ones, theorically they dont break stuff on patch versions